contracts/market/template/MarketExtended.sol

```diff
@@ -132,13 +149,13 @@ contract MarketExtendedCore is AccessControlledAndUpgradeableModifiers, MarketSt
     );
 
     SinglePoolInitInfo memory poolInfo = initPool;
-    uint256 tierPriceMovementThresholdAbsolute = PoolType.FLOAT == initPool.poolType ? 0.1999e18 : (1e36 / poolInfo.leverage) - 1e14;
+    uint256 tierPriceMovementThresholdAbsolute = PoolType.FLOAT == initPool.poolType ? 0.19e18 : (1e36 / poolInfo.leverage) - 1e16;
 
     maxPercentChange = int256(Math.min(uint256(maxPercentChange), tierPriceMovementThresholdAbsolute));
 
     IPoolToken(initPool.token).initialize(initPool, seederAndAdmin, _marketIndex, uint8(_numberOfPoolsOfType[uint256(initPool.poolType)]));
 
-    IPoolToken(initPool.token).mint(MARKET_SEEDER_DEAD_ADDRESS, initialActualLiquidityForNewPool);
+    IPoolToken(initPool.token).mint(MARKET_SEEDER_DEAD_ADDRESS, MathUintFloat.div(initialActualLiquidityForNewPool, initialPoolTokenPrice));
```

Comment:
Factor in the initial pool price when minting the initial pool tokens on initialization. This sets their starting price.